_schemaV2.graphql

```diff
@@ -3168,6 +3168,13 @@ enum ArtworkSorts {
 
 # An artwork rail section in the home view
 type ArtworksRailHomeViewSection implements GenericHomeViewSection & Node {
+  artworksConnection(
+    after: String
+    before: String
+    first: Int
+    last: Int
+  ): ArtworkConnection
```

Comment:
We discussed the merits of

-  having a simple `artworksConnection` here that takes **basic connection arguments only** as shown here, and letting each home view artworks resolver be more opinionated about what [ultimate args](https://github.com/artsy/metaphysics/pull/5865/files#diff-294b344a66a8ab3968329ebdf23ee095eaa754562be805266f228cd2ac4be3dfR18-R28) and ultimate [resolver should](https://github.com/artsy/metaphysics/pull/5865/files#diff-294b344a66a8ab3968329ebdf23ee095eaa754562be805266f228cd2ac4be3dfR30-R35) be used
    - works well for the paradigm where the _server is the authority_ about what objects should be shown in home view

-versus-

- trying to accommodate many other types of args that could be used by the various kinds of `ArtworksRailHomeViewSection` — e.g. NEW_WORKS_FOR_YOU, RECENTLY_VIEWED_ARTWORKS, etc etc
  - would _allow the client to influence_ what objects should be shown in home view

-versus-

- actually providing multiple, separate artwork fields for different section types
  - can't remember the advantage of this, but I think we agreed it would gunk up the schema